### ISSUE TITLE
A11y_AzureDataStudio_ClusterSettings_ScreenReader:NVDA does not announces the "Basic" combo box while navigation on the screen.

### ISSUE THREAD

**Issue by vinodbk197** [microsoft/azuredatastudio#7665]
"[Check out Accessibility Insights!](https://accessibilityinsights.io) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

**GitHubTags:**#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#SQL Azure Data Studio;#WCAG1.3.1;#WCAG2.4.7;#DesktopApp;#NVDA;#JAWS;#Win32;

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 32df727
Date: 2019-10-08T19:18:43.743Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS: Windows_NT x64 10.0.18362

**Additional Details:**
NVDA Version:2019.2.1

**Repro Steps:**
1.	Install /Launch the application Azure data studio.
2.	Navigate to connections and click on more action button.
3.	Select “Deploy SQL Server big data cluster” from more action drop down.
4.	Click on “Deploy SQL Server big data cluster”.
5.	Check the check box I accept in the “select the deployment option” screen and click on select button.
6.	Select any block and click on next button and again click on next button.
7.	Start the NVDA screen reader and navigate on the screen. 
9.   Check the screen reader focus is  going on "Basic" combo box or not.

**Actual:**
NVDA screen reader focus is not going on the "Basic" combo box while navigating on the screen.Here after "Authentication mode" screen reader focus moves to "previous" button.

**Expected:**
NVDA screen reader focus should go on the "Basic" combo box while navigating on the screen and it announces the same.

**User Impact:**
Screen reader user will does not get all information available in the screen if screen reader focus not moves to the control.

**Attachment for Reference:**
[7665_A11y_AzureDataStudio_ClusterSettings_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3717034/7665_A11y_AzureDataStudio_ClusterSettings_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by alanrenmsft** [comment]
this is because the dropdown itself is disabled and it shouldn't be focusable. @vinodbk197  please let me know if you think disabled control should be focusable.

**Comment by vinodbk197** [comment]
Verified the issue on:
Azure Data Studio
Version: 1.15.0-insider (user setup)

The issue is resolved,Hence closed this issue.
Please refer the below attachment:
![image](https://user-images.githubusercontent.com/55909580/72731332-88e63c00-3bb9-11ea-9b78-707d0ab8b2bc.png)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_win32;
